### ISSUE TITLE
Scylla setup 3.1 on GCP not clean :(, should not spit out such error messages, and halt forever

### ISSUE THREAD

**Issue by eyalgutkind** [scylladb/scylla#5306]
*Installation details*
Scylla version (or git commit hash):3.1.1-0.20191024.f32ec885c
Cluster size: 1 
OS (RHEL/CentOS/Ubuntu/AWS AMI): Centos 7
```
[eyal@eg-test31 ~]$ cat /etc/*rel*
CentOS Linux release 7.7.1908 (Core)
Derived from Red Hat Enterprise Linux 7.7 (Source)
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

cat: /etc/prelink.conf.d: Is a directory
CentOS Linux release 7.7.1908 (Core)
CentOS Linux release 7.7.1908 (Core)
cpe:/o:centos:centos:7
[eyal@eg-test31 ~]$ uname -a
Linux eg-test31 3.10.0-1062.4.3.el7.x86_64 #1 SMP Wed Nov 13 23:58:53 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```
*Hardware details (for performance issues)*          Delete if unneeded
Platform (physical/VM/cloud instance type/docker): GCP, n1-standard-16, 100GB SSD normal drive.
```
[eyal@eg-test31 ~]$ df -Th
Filesystem     Type      Size  Used Avail Use% Mounted on
devtmpfs       devtmpfs   30G     0   30G   0% /dev
tmpfs          tmpfs      30G     0   30G   0% /dev/shm
tmpfs          tmpfs      30G  8.5M   30G   1% /run
tmpfs          tmpfs      30G     0   30G   0% /sys/fs/cgroup
/dev/sda1      xfs       100G  2.3G   98G   3% /
tmpfs          tmpfs     5.9G     0  5.9G   0% /run/user/1023
```
The following errors should not be part of a normal installation using Scylla setup on a vanilla machine and OS

```
[eyal@eg-test31 ~]$ sudo scylla_setup
Skip any of the following steps by answering 'no'
Do you want to run check your kernel version?
Yes - runs a  script to verify that the kernel for this instance qualifies to run Scylla. No - skips the kernel check.
[YES/no]
**WARN  2019-11-19 10:38:43,874 [shard 0] iotune - Available space on filesystem at /var/tmp/mnt: 124 MB: is less than recommended: 10 GB
INFO  2019-11-19 10:38:43,875 [shard 0] iotune - /var/tmp/mnt passed sanity checks
This is a supported kernel version.**
```
this one is funny :)
```
Created symlink from /etc/systemd/system/multi-user.target.wants/scylla-server.service to /usr/lib/systemd/system/scylla-server.service.
Do you want to enable Scylla to check if there is a newer version of Scylla available?
Yes - start the Scylla-housekeeping service to check for a newer version. This check runs periodically. No - skips this step.
[YES/no]
**You current Scylla release is 3.1.1 the latest minor release is 3.0.8 go to http://www.scylladb.com for upgrade instructions**
Do you want to disable SELinux?
Yes - disables SELinux. Choosing Yes greatly improves performance. No - keeps SELinux activated.
```

and now this...

```
Yes - let iotune study my disk(s). Note that this action will take a few minutes. No - skip this step.
[YES/no]
**/usr/lib/scylla/libexec/scylla_io_setup:88: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))**
tuning /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:1/0:0:1:0/block/sda/sda1
tuning /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:1/0:0:1:0/block/sda
tuning: /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:1/0:0:1:0/block/sda/queue/nomerges 2
tuning /sys/devices/pci0000:00/0000:00:03.0/virtio0/host0/target0:0:1/0:0:1:0/block/sda/sda1
```
And scylla_setup is halted forever. Am I doing something wrong?

**Comment by slivne** [comment]
iotune hang just reproduced https://github.com/scylladb/scylla/issues/5151 thay avi and glauber looked at - thanks

**Comment by tzach** [comment]
> You current Scylla release is 3.1.1 the latest minor release is 3.0.8 

@amnonh FYI

**Comment by avikivity** [comment]
Closed as duplicate of #5151.
